core/strconv/strconv.odin

```diff
@@ -47,18 +47,19 @@ parse_i64_of_base :: proc(str: string, base: int) -> (value: i64, ok: bool) {
 
 	i := 0;
 	for r in s {
-		i += 1;
-		if r == '_' do continue;
-		v := i64(_digit_value(r));
-		if v >= i64(base) {
-			break;
+		if r == '_' {
+			i += 1;
+			continue;
 		}
+		v := i64(_digit_value(r));
+		if v >= i64(base) do break;
 		value *= i64(base);
 		value += v;
+		i += 1;
 	}
 
 	if neg do value = -value;
-	ok = i > 1;
+	ok = i > 0;
```

Comment:
For clarity, I:
- Moved `i += 1;` to the bottom of the loop (when a valid digit is processed)
- Added another `i += 1;` for when it's an underscore
- Have it return `ok=true` if `i > 0`.